logxhost/db.go

```diff
@@ -4,14 +4,30 @@ import (
 	"errors"
 
 	"github.com/Masterminds/squirrel"
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+	errs "github.com/pkg/errors"
 )
 
 var psql = squirrel.StatementBuilder.PlaceholderFormat(squirrel.Dollar)
 
 const (
 	TableService = "service"
+	TableLogs    = "log_view"
```

Comment:
Should be `ViewLog`

Reply:
We should also have a `TableLog` for updating and inserting logs into the table. Probably my bad.